src/CMakeLists.txt

```diff
@@ -225,6 +225,9 @@ set (server_sources
      ${PROJECT_SOURCE_DIR}/third_party/lua-yaml/lyaml.cc
      ${PROJECT_SOURCE_DIR}/third_party/lua-cjson/lua_cjson.c
      ${PROJECT_SOURCE_DIR}/third_party/lua-cjson/strbuf.c
+     ${PROJECT_SOURCE_DIR}/third_party/nanoarrow/flatcc.c
+     ${PROJECT_SOURCE_DIR}/third_party/nanoarrow/nanoarrow.c
+     ${PROJECT_SOURCE_DIR}/third_party/nanoarrow/nanoarrow_ipc.c
```

Comment:
Can we build it as a separate library, like zstd?

Reply:
Done.